src/Components/Results/211Button/211Button.tsx

```diff
@@ -5,7 +5,7 @@ const HelpButton = () => {
   return (
     <div className="help-text-for-211-button">
       <h2 className="text-center help-text-for-211-button-font">{`Can't find what you need?`}</h2>
-      <div>
+      <button className="hover-button">
```

Comment:
This might be fixed by pulling from `feat/result_redesign`, but can you make this back into a `div`